polaris-react/src/components/Filters/components/FiltersBar/FiltersBar.tsx

```diff
@@ -177,7 +173,7 @@ export function FiltersBar({
           disableFilters
         }
       >
-        <Text variant={labelVariant} as="span">
+        <Text as="span" variant="bodySm">
```

Comment:
Without the `color: var(--p-text-color)` styles the color of the Add Filter button is inconsistent with the other filters. 
![20-33-eg1lt-jm6jj](https://github.com/Shopify/polaris/assets/26749317/83eb6c4a-f3f0-45de-ba9c-f51b52b480ec)

Reply:
Just for context, Text has never had a default tone/color because our legacy typography components didn't and we tried to keep that consistent during the typography project 👀

If we want to set it with `tone`, I'm leaning towards `tone="base"` because it's consistent with Icon

Reply:
There are a couple of issues open around `base` but this sounds fine to me.
https://github.com/Shopify/ui-api-design/issues/317
https://github.com/Shopify/ui-api-design/issues/301

Reply:
Good catch. I've added a `disabled` option to tone and updated the prop logic for FiltersBar